scipy-1.0/subpackages.tex

```diff
@@ -47,20 +45,20 @@
     and functions to create specialized matrices, such as diagonal, Toeplitz,
     Hankel, companion, Hilbert, and more.
 \item[\texttt{ndimage}]
-    This package contains various functions for multi-dimensional image
+    This subpackage contains various functions for multi-dimensional image
     processing, including convolution and assorted linear and nonlinear
     filters (Gaussian filter, median filter, Sobel filter, etc.);
     interpolation; region labeling and processing; and mathematical morphology
     functions.
 \item[\texttt{misc}]
     A collection of functions that did not fit into the other subpackages.
     While this subpackage still exists in SciPy 1.0, effort is underway
-    to deprecate or relocate the contents of this subpackage and eventually remove it.
```

Comment:
I removed this word purely for formatting: "it" was by itself on the next line.  I don't think this is inaccurate. Work is underway to deprecate/relocate its contents and remove it.